plugins/chat/assets/javascripts/discourse/components/chat-message-info.js

```diff
@@ -31,9 +32,6 @@ export default class ChatMessageInfo extends Component {
     if (user.moderator) {
       classes.push("is-moderator");
     }
-    if (user.groupModerator) {
```

Comment:
removed this one for now, as the use case seems low and it would be adding more overhead